src/widgets/QDisassemblyView.cpp

```diff
@@ -1285,38 +1286,85 @@ void QDisassemblyView::paintEvent(QPaintEvent *) {
 			return a.distance < b.distance;
 		});
 
-		int width_upward_outviewport = 0;
-		int width_downward_outviewport = 0;
+		// find suitable arrow horizontal width
+		for (int i = 0; i < jump_arrow_vec.size(); i++) {
```

Comment:
I'd prefer the `i`, `j` and `k` variables to be named more descriptively. It's hard to keep in mind what three one-letter variables represent when trying to follow the logic.

Reply:
I've renamed all these three variables.